### ISSUE TITLE
feature request: c++ support

### ISSUE THREAD

**Issue by Tokusei** [Chentai-Kao/codemap-public#3]
hi, are there any plans to add dependency visualization for C++ files?

**Comment by Chentai-Kao** [comment]
Yes, support for other languages is coming in the future. My current plan is to build support for Java, C/C++

**Comment by gevorgyana** [comment]
Can I help you with that? I am also interested in Rust. I think based on the fact they share common backend with C++, the implementation should be similar in a way.
